plugins/redux/extend.js

```diff
@@ -0,0 +1,18 @@
+const base = {
+  _app: {
+    import: [
+      "import React from 'react'",
```

Comment:
```suggestion
```
bence bunu template'in app'ine ekleyelim

Reply:
tamamdır bunu ben ekliyorum